nn_surrogate_benchmark/surrogate_sobol.py

```diff
@@ -0,0 +1,209 @@
+import torch
+import torch.nn as nn
+import pytorch_lightning as pl
+from torch.optim import AdamW, Adam
+from typing import Literal
+import pandas as pd
+import numpy as np
+from torch.utils.data import TensorDataset, DataLoader
+from sklearn.preprocessing import (
+	StandardScaler,
+	MinMaxScaler,
+	RobustScaler,
+	MaxAbsScaler,
+)
+
+class Sobolev(pl.LightningModule):
+	def __init__(
```

Comment:
This formatting differs from my default; I need to properly set up pre-commit.

Reply:
Ok, for the moment I tried unifying the formatting